proto/process/agent.proto

```diff
@@ -85,6 +87,10 @@ message CollectorConnections {
 	string platform = 19;
 	string platformVersion = 20;
 
+	// host configuration, ephemeral port range
```

Comment:
Any reason for not putting these at the end?

Reply:
they seemed to fit together with bytes sent/received.